apps/evm/lib/evm/message_call.ex

```diff
@@ -155,6 +155,27 @@ defmodule EVM.MessageCall do
     end
   end
 
+  defp finalize_failed_message_call(
+         {_remaining_gas, _n_sub_state, _exec_env, :failed},
+         message_call
+       ) do
+    result = failed_call(message_call)
+
+    if message_call_to_rip_md?(message_call) do
+      sub_state =
+        message_call.current_sub_state
+        |> SubState.add_touched_account(message_call.recipient)
+
+      Map.put(result, :sub_state, sub_state)
+    else
+      result
+    end
+  end
+
+  defp message_call_to_rip_md?(message_call) do
+    message_call.recipient == <<3::160>>
```

Comment:
can we move `<<3::160>>` to module attribute or maybe to the module with rip builtin?

Reply:
Yeah, very good idea.

Reply:
Updated in 1150407669ec07803c7d8615524f2d4a9a9c62ec